source/authentication/api-key.txt

```diff
@@ -31,6 +31,8 @@ associates that API key with an automatically created {+service-short+} server u
 Provide a server key to external applications and services to allow them
 to authenticate directly with {+service-short+}.
 
+You can associate up to 100 server keys with an app. 
```

Comment:
nit 

```suggestion
You can associate up to 100 server API keys with an app. 
```

Reply:
Updated this and the below section for consistency. Thanks!